action.yml

```diff
@@ -57,6 +57,12 @@ inputs:
     required: false
     type: boolean
     default: 'true'
+  artifact-path:
```

Comment:
what if this is already a zip?

Reply:
I added a check, if the file is already a zip file the procedure won't zip it.